### ISSUE TITLE
Django 1.9 not support url data

### ISSUE THREAD

**Issue by kurkop** [aljosa/django-tinymce#145]
This error occurs when I add: url(r'^tinymce/', include('tinymce.urls')) to urls.py (urlpatterns).
```
urlpatterns = [
    url(r'^admin/', admin.site.urls),
    url(r'^$', views.home, name='home'),
    url(r'^tinymce/', include('tinymce.urls')),
]
```
```
/usr/local/lib/python3.4/site-packages/tinymce/urls.py:10: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got textareas_js). Pass the callable instead.
  url(r'^js/textareas/(?P<name>.+)/$', 'textareas_js', name='tinymce-js'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:11: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got textareas_js). Pass the callable instead.
  url(r'^js/textareas/(?P<name>.+)/(?P<lang>.*)$', 'textareas_js', name='tinymce-js-lang'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:12: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got spell_check). Pass the callable instead.
  url(r'^spellchecker/$', 'spell_check'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:13: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got flatpages_link_list). Pass the callable instead.
  url(r'^flatpages_link_list/$', 'flatpages_link_list'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:14: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got compressor). Pass the callable instead.
  url(r'^compressor/$', 'compressor', name='tinymce-compressor'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:15: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got filebrowser). Pass the callable instead.
  url(r'^filebrowser/$', 'filebrowser', name='tinymce-filebrowser'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:16: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got preview). Pass the callable instead.
  url(r'^preview/(?P<name>.+)/$', 'preview', name='tinymce-preview'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:16: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^preview/(?P<name>.+)/$', 'preview', name='tinymce-preview'),
```

**Comment by hugodes** [comment]
+1

**Comment by jccode** [comment]
+1

**Comment by Natim** [comment]
Feel free to file a PR to fix this I will look at it afterwards.

**Comment by hugodes** [comment]
@Natim made a pull request to master : #148 . I hope it was the right branch to do so. Please tell me if otherwise

**Comment by Natim** [comment]
Released with in django-tinymce 2.1.0
